acceptance/tests/utf8/utf8-in-function-args.rb

```diff
@@ -14,6 +20,7 @@
         result = on(
           agent,
           puppet("apply", "-e" "'alert(\"alert #{utf8chars}\")'"),
+          :environment => {:LANG => "en_UTF-8"}
```

Comment:
@seangriff I think we want these to be `:LANG => "en_US.UTF-8"`? At least on Ubuntu 16.04 that's the case:

```ruby
☂ : LANG="en_UTF-8" ruby -e "puts Encoding.default_external"
US-ASCII
☂ : locale-gen "en_UTF-8"
Error: 'en_UTF-8' is not a supported language or locale
☂ : locale-gen "en_US.UTF-8"
Generating locales (this might take a while)...
  en_US.UTF-8... done
Generation complete.
☂ : LANG="en_US.UTF-8" ruby -e "puts Encoding.default_external"
UTF-8
```

Down below later in the test it looks like you do indeed switch over to `en_US.UTF-8` ie, line 62 and later